tasks/opensuse/install.yml

```
@@ -0,0 +1,21 @@
+---
+- name: OpenSUE | Install Repo
+  become: true
+  community.general.zypper_repository:
+    repo: "{{ opensuse_tumbleweed_repository }}"
+    state: '{{ state }}'
+  when: ansible_distribution == "openSUSE Tumbleweed"
+
+- name: OpenSUSE | Install Repo
```

To make the output more readable:

```suggestion
- name: OpenSUSE | Install Repo - Leap
```